flake.nix

```diff
@@ -38,12 +38,12 @@
         haskellPackages.override {
           overrides = self: super: {
             hpack = pkgs.hpack;
-            crem = (self.callCabal2nix "crem" src { }).overrideAttrs (attrs: {
+            crem = pkgs.haskell.lib.compose.dontCheck ((self.callCabal2nix "crem" src { }).overrideAttrs (attrs: {
```

Comment:
what is the effect of this?

When you build the library with nix:
* it does not run the tests anymore
* the build does not fail if there is an compile time error in the tests